aff4/aff4_map.cc

```diff
@@ -606,7 +606,7 @@ void AFF4Map::Dump() {
 }
 
 
-std::vector<Range> AFF4Map::GetRanges() {
+std::vector<Range> AFF4Map::GetRanges() const {
```

Comment:
Not sure how often this method is called or how large `map` can get, but, @uckelman-sf, it looks like it would be a simple one-liner to add `result.reserve(map.size());` so that `result`'s allocation was correct from the start.

It's possible to replace the for-loop with use of `std::transform`, `std::back_inserter`, and a lambda, but I think this is more readable, tbh.

Reply:
I think this comment is stale now following some changes by @jtsylve.